src/styles/fonts.tsx

```diff
@@ -40,8 +41,110 @@ const standards = {
     color: colors.dark,
   },
 }
-// Figma Font Styles
-const fontStyles = StyleSheet.create({
+
+/**
+ * Figma TypeScale Styles
+ */
+export const typeScale = StyleSheet.create({
+  displayLarge: {
+    fontFamily: Inter.Bold,
+    fontSize: 80,
+    lineHeight: 80,
+    letterSpacing: -3,
```

Comment:
Looking at the figma file and selecting this one, it says `letter-spacing: -2.4px;` as CSS prop. Which is 3% from 80 (the line height).

<img width="1102" alt="Screenshot 2023-09-20 at 10 33 38" src="https://github.com/valora-inc/wallet/assets/57791/6b6de150-805a-45f5-a762-9800bfb12c5a">

```suggestion
    letterSpacing: -2.4,
```

Similar adjustments are needed for the other styles below.

Reply:
Good catch I'll make this adjustment!

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/4211/commits/9dc5d3e480cbd810c3c23eaea219f22401219f7c!